docs/documentation/github-desktop.md

```diff
@@ -0,0 +1,38 @@
+# Store your code online with GitHub and GitHub Desktop
+
+GitHub is a way to store code online so you can collaborate with other people. It also makes it easier to publish your prototype online with Heroku.
+
+Some concepts:
+
+- Git - a way to store versions of your code
+- GitHub - a website that shares your code online
+- GitHub Desktop - an app that puts your code on GitHub
+- Repository/repo - a project that has been stored in Git
+
+## Set up GitHub Desktop
+
+1. Create a GitHub account, it's free: https://github.com/join
+
+2. Download GitHub Desktop from https://desktop.github.com/
+
+3. Run GitHub Desktop, sign in with your GitHub account details.
```

Comment:
Is this one step or two? If it's two, we could say:

"3.  Run GitHub Desktop.

4. Sign in with your GitHub account details."